.gitattributes

```diff
@@ -4,6 +4,11 @@
 * text=auto
 
 ###############################################################################
+# Explicitly declare files that should always be converted to LF regardless of platform
+###############################################################################
+*.dotsettings text eol=lf
```

Comment:
`.dotsettings` files are the files where we can configure ReSharper settings for the solution. These files should always have Unix line-endings, even on Windows, so this is what this does... Make sure `.dotsettings` files are always converted to LF when you clone the repository / checkout a branch.